adcc/ExcitedStates.py

```diff
@@ -657,6 +659,48 @@ def describe_amplitudes(self, tolerance=0.01, index_format=None):
                 ret += separator
         return ret[:-1]
 
+    def to_dataframe(self, export_vectors=True):
```

Comment:
Can you think of any case where you would want `export_vectors=False`?

Reply:
Say you're using a notebook and want to display the dataframe there... you might not want a df that runs too wide...

Reply:
Yes but then isn't it just a question of ordering the important columns first?

Reply:
Well... what is "important"? 😄 Might be different for everyone...
I'd say we drop the extra flag, such that you always get everything. you can then of course reduce the table on your own if you want to.